Source/Charts/Formatters/IValueFormatter.swift

```diff
@@ -11,11 +11,11 @@
 
 import Foundation
 
-/// Interface that allows custom formatting of all values inside the chart before they are being drawn to the screen.
+/// Interface that allows custom formatting of all values inside the chart before they are drawn to the screen.
 ///
-/// Simply create your own formatting class and let it implement ValueFormatter.
-///
-/// Then override the getFormattedValue(...) method and return whatever you want.
+/// Simply create your own formatting class and let it implement ValueFormatter. Then override the stringForValue()
+/// method and return whatever you want.
```

Comment:
I'm not sure if this is github display issue or your code. `/// method and return whatever you want.` in a new line is strange to me

Reply:
Fixed.